examples/ssd_mobilenet/utils.js

```diff
@@ -139,316 +135,4 @@ class Utils {
       }
     }
   }
-
-  /**
-  * Decode out box coordinate
```

Comment:
why move those functions to base.js. I suppose those are only used by SSD example. Correct?

Reply:
I will reuse these functions for benchmark test with SSD MobileNet, so move them to base.js.thanks